app/controllers/corporate_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CorporateController < ApplicationController
+  decorates_assigned :corporate, with: ContentItemDecorator
+  include Navigation
+
+  def show
+    @corporate = Core::CorporateReader.new(params[:id]).call do
+      not_found
+    end
+
+    assign_active_categories(*@corporate.categories)
```

Comment:
@dougdroper I was in doubt what categories should I display here in corporate. I put this, at least for now.